lib/cosmos/tools/script_runner/script_runner_frame.rb

```diff
@@ -752,6 +752,14 @@ def comment_or_uncomment_lines
       @script.comment_or_uncomment_lines unless running?()
     end
 
+    def zoom_in
+      @script.zoom_in unless running?()
```

Comment:
Any reason not to support this while running?

Reply:
Its jarring to have this happen when running an active script.

Reply:
I can see wanting to zoom without having to stop the script...